docs/workers.md

```diff
@@ -1,670 +0,0 @@
-# Scaling synapse via workers
```

Comment:
you seem to have deleted this entire file?

Reply:
Whoops, sorry about that. Still getting used to GitHub. Hopefully it's fine now!